src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -3,72 +3,130 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Threading.Tasks;
 using Microsoft.DotNet.ProjectModel;
```

Comment:
Not yet, definitely something to look into. https://github.com/aspnet/BuildTools/issues/127